libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -41,6 +44,16 @@ export class BookSearchComponent {
 
   addBookToReadingList(book: Book) {
     this.store.dispatch(addToReadingList({ book }));
+    // show snack bar message with undo action
+    const snackBarRef = this.snackBar.open('Added "' + book.title + '" to reading list.', 'Undo', { duration: 10000});
```

Comment:
What happens if the `addToReadingList` action fails?

Shouldn't we be triggering this on the confirm action?

Reply:
I agree, this should be part of successful add action. Handling the snackBar message actions in `addBook$` and `removeBook$` effects.